module/Makefile

```diff
@@ -0,0 +1,26 @@
+# find the OS
+uname_S := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+
+# Compile flags for linux / osx
+ifeq ($(uname_S),Linux)
+	SHOBJ_CFLAGS ?= -W -Wall -fno-common -g -ggdb -std=c99 -O2
+	SHOBJ_LDFLAGS ?= -shared
+else
+	SHOBJ_CFLAGS ?= -W -Wall -dynamic -fno-common -g -ggdb -std=c99 -O2
+	SHOBJ_LDFLAGS ?= -bundle -undefined dynamic_lookup
+endif
+
+.SUFFIXES: .c .so .xo .o
+
+all: trie.so
+
+.c.xo:
```

Comment:
What is a `.xo` file and why are we using it?

Reply:
Good point this actually isn't necessary. I saw this in a Makefile for another Redis module and thought it was critical at first (wasn't sure why) but I realized this is just an extra step. I made changes in the Makefile to reflect this so there's no .xo file.

Reply:
I talked to Ariel as well and she said it's definitely unnecessary.